core/metric_registry.go

```diff
@@ -117,7 +125,7 @@ func NewCommonMetricSampler(registry MetricRegistry, limit Limit, name string, t
 // Sample will sample the current sample for metric reporting.
 func (s *CommonMetricSampler) Sample(rtt int64, inFlight int, didDrop bool) {
 	if didDrop {
-		s.InFlightListener.AddSample(1.0)
+		s.DropCounterListener.AddSample(1.0)
```

Comment:
Not 100% sure but I think this was the reason why I wouldn't see any drop metrics outputted.